index.html

```diff
@@ -20,34 +20,166 @@
-            </svg></a></li>
-          </ul>
-        </nav>
-    </section>
+        </svg></a></li>
+      </ul>
+    </nav>
+  </section>
+  <section class="work">
+    <div class="card card1 img1">
+      <img src="./Img Plaholder.png" alt="">
+      <h2 class="paragraph_card">Multi-Post Stories <br>Gain+Glory</h2>
+      <div class="stack">
+        <ul class="listw">
+          <li ><div class="ruby">Ruby on rails</div></li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+      </div>
+      <a href="#" target="_blank" id="button"><p>See Project</p></a>
+    </div>
+    <div class="card card2 img1">
+      <img src="./Img Plaholder.png" alt="">
+      <h2 class="paragraph_card">Multi-Post Stories <br>Gain+Glory</h2>
+      <div class="stack">
+        <ul class="listw">
+          <li ><div class="ruby">Ruby on rails</div></li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+      </div>
+      <a href="#" target="_blank" id="button"><p>See Project</p></a>
+    </div>
+    <div class="card ard3 img1">
+      <img src="./Img Plaholder.png" alt="">
+      <h2 class="paragraph_card">Multi-Post Stories <br>Gain+Glory</h2>
+      <div class="stack">
+        <ul class="listw">
+          <li ><div class="ruby">Ruby on rails</div></li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+      </div>
+      <a href="#" target="_blank" id="button"><p>See Project</p></a>
+    </div>
+    <div class="card card4 img1">
+      <img src="./Img Plaholder.png" alt="">
```

Comment:
- Name the image files in `lowercase_with_underscores.png` 
- And put all the images in `img` folder. Few are there in `img`. Few are at `root` of project. Few are being used as `svg`s in HTML file. Place them all in one place and import them from there.